### PR TITLE
feat: Add page for chanbakjsd

### PR DESCRIPTION
**Pull request by chanbakjsd**

### CODE REVIEW

File: pages/chanbakjsd.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Click here for [Docs](https://docs.ntuscse.com)!
```
Comment (ntu-scse-it) [inline pages/chanbakjsd.md:1]: let's use lowercase in docs

**Review by ntu-scse-it — CHANGES_REQUESTED** [review]
See inline comments

File: pages/chanbakjsd.md (hunk 1 above)
Comment (chanbakjsd) [reply pages/chanbakjsd.md]: The changes have been made.

**Review by teamoteh — APPROVED** [review]
Nice work Chanbakjsd

**Review by kingsmil — CHANGES_REQUESTED** [review]
add a fun fact

**Comment by teamoteh** [comment]
> add a fun fact

**Comment by aryans1204** [comment]
LGTM!!

File: pages/chanbakjsd.md (hunk 1 above)
Comment (ntu-scse-it) [reply pages/chanbakjsd.md]: OKIE 🚀

**Review by ntu-scse-it — APPROVED** [review]
LGTM 👍

**Pull request merged**